tests/testdata/codegen/azure-native-pp/go/azure-native.go

```diff
@@ -27,30 +27,30 @@ func main() {
 		_, err = cdn.NewEndpoint(ctx, "endpoint", &cdn.EndpointArgs{
 			Origins: cdn.DeepCreatedOriginArray{},
 			DeliveryPolicy: &cdn.EndpointPropertiesUpdateParametersDeliveryPolicyArgs{
-				Rules: []cdn.DeliveryRuleArgs{
-					{
+				Rules: cdn.DeliveryRuleArray{
```

Comment:
How come this got updated?

Reply:
I think it had the wrong type because we were lowering expressions twice 🤔